### DISCUSSION TITLE
Environment app updates

### DISCUSSION THREAD

**Discussion by lloydbayley** [Nachtzuster/BirdNET-Pi#192] [General]
How are we managing environmental app updates?
Eg: Apprise, Python etc etc

For instance, I raised an issue and we merged a PR on Apprise and added the retain flag to MQTT publishes.
I'm now testing this in my environment to allow for the retention of the Home-Assistant setup card data after a reboot.

I wonder if it's better, as happens at the moment, to manually update the update file with the next version only if the need arises, so as to prevent accidental breakages. I wouldn't be so concerned about things like Apprise, but with Python, it could be a fundamental disaster.

Thoughts?

**Comment by Emmo213** [comment]
In a previous PR I tried to update the python version and was told we couldn't for those who migrated from the original project. At some point we'll want more current versions but there wasn't a proposed solution.

**Reply by lloydbayley** [reply]
Yeh, I shouldn't have said Python in that bit, that's another (scary) creature/type...I meant the stuff that update_birdnet_snippets.sh takes care of...moreso streamlit, apprise and seaborn etc
Although a plan for python might be an idea too.
